composer.json

```diff
@@ -23,23 +23,28 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "yiisoft/yii-console": "^3.0@dev",
-        "yiisoft/strings": "^3.0@dev",
+        "php": "^7.4",
+        "psr/event-dispatcher": "^1.0",
+        "symfony/process": "^4.0",
         "yiisoft/mutex": "^3.0@dev",
         "yiisoft/serializer": "^3.0@dev",
-        "symfony/process": "^4.0",
-        "php": "^7.4"
+        "yiisoft/strings": "^3.0@dev"
     },
     "require-dev": {
+        "ext-pdo": "*",
+
+        "aws/aws-sdk-php": ">=2.4",
         "enqueue/amqp-lib": "^0.9",
```

Comment:
It is used in `src`. Likely it worth moving it and all related code to separate driver-package later.

Reply:
Yes, all drivers will be implemented separately, as I've already said above.